playground/bicep/BicepSample.AppHost/aspire.hosting.azure.bicep.webpubsub.bicep

```diff
@@ -0,0 +1,73 @@
+param name string
+param principalId string
+
+@description('Tags that will be applied to all resources.')
+param tags object = {}
+
+@description('The pricing tier of the resource.')
+@allowed([
+  'Free_F1'
+  'Standard_S1'
+  'Premium_P1'
+])
+
+param pricingTier string = 'Free_F1'
+
+@description('The number of the Unit.')
```

Comment:
What does "The number of the Unit." mean?

Reply:
![image](https://github.com/dotnet/aspire/assets/668244/7ba4004e-babf-474e-9c79-129f5b8a58d3)

Reply:
update to use unit count